rpc/openapi/openapi.yaml

```
@@ -911,6 +911,39 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  /genesis_chunked:
+    get:
+      summary: Get Genesis in multiple chunks
+      operationId: genesis_chunked
+      tags:
+        - Info
+      description: |
+        Get genesis document in multiple chunks to make it
+        easier to iterate through larger gensis structures.
+
+        Upon success, the `Cache-Control` header will be set with the default
+        maximum age.
+      parameters:
+        - in: query
+          name: chunkID
```

Looking at https://github.com/tendermint/tendermint/blob/99a7ac84dca30676fd544be18c6df2880a14429f/rpc/core/routes.go#L22 shouldn't this be:

```suggestion
          name: chunk
```
?